File: unit2/arg_main.c
#include <stdio.h>
#include <stdlib.h> // atoi, atof//
#include <string.h> // strcpy //

int my_int = 0;           //%i o %d
float my_float = 3.14;    // %f
char my_char = 'c';       // %c
char my_str[10] = "Hola"; // %s

float second_float = 0.0;
// pointer //
int *my_ptr_int = &my_int; // %p
//float* my_ptr_float = &second_float;

int main(int argc, char **argv)
{
    //Pegando el argv1 a la var string
    strcpy(my_str, argv[1]);
    // Convertir string a entero
    int base = atoi(my_str);
    //printf("num %i", num*2);
    strcpy(my_str, argv[2]);
    int altura = atoi(my_str);

    printf("my_int: %i my_float: %f, my_char: %c, my_str: %s, my_ptr_int: %p \n", my_int, my_float, my_char, my_str, my_ptr_int);
    printf("argc: %i \nelement 1: %d \nelement 2: %s \nchar: %c \n ", argc, base * altura, argv[2], 'L');

    return 0;
}

File: unit2/arg_main.py
import sys

# Variable Globales##
my_int = 0
my_str = ""
my_float = 0.0
my_bool = True  # false
my_list = []  # list

if __name__ == "__main__":

    my_int = int(sys.argv[1])
    my_str = str(sys.argv[2])  # "Hola mundo"
    my_float = float(sys.argv[3])  # 3.14
    my_bool = bool(sys.argv[4])  # False
    my_list = sys.argv  # [1,2,3,4,5] ##list()
    # 0 1 2 3 4

    print(my_int, my_int * 2, my_str, my_float, my_bool, "|", my_list)
    # print("Name program:", sys.argv[1])
